src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,33 @@
+import org.junit.jupiter.api.Test;
+import java.time.LocalDate;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void deadlineConstructorTest1(){ //tests basic constructor
```

Comment:
Do you think it would be a better idea to give the test a more descriptive name? E.g. `whatIsBeingTested_descriptionOfTestInputs_expectedOutcome`. This applies to the other tests as well.